check_manifest.py

```diff
@@ -605,6 +604,18 @@ def read_config():
         IGNORE_BAD_IDEAS.extend(p for p in patterns if p)
 
 
+def _find_config(config):
+    """Search setup.cfg and tox.ini for the section '[check-manifest]'.
+    If successful, the file will have been read into the config object."""
```

Comment:
How about 

```suggestion
    """Search for a config file and read it.
    
    Looks for a ``check-manifest`` section in ``setup.cfg`` and ``tox.ini``, in that order.
    The first file that exists and has that section will be loaded into `config`, which
    should be a ConfigParser instance.
    
    Returns ``True`` if a config file with check-manifest configuration was found,
    ``False`` otherwise.
    """
```
?